_posts/2021-04-09-prace-workshop.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: MDAnalysis Workshop @ PRACE / SURF
+---
+
+MDAnalysis will be delivering a 3 half-day remote (online) workshop hosted by [PRACE](https://prace-ri.eu/) / [SURF](https://www.surf.nl/en) on the 26-28th of May 2021.
+
+The workshop will include both lecture and practical portions. We will [cover topics](https://events.prace-ri.eu/event/1201/timetable/) including:
+
+* The fundamentals and basics of MDAnalysis
+* How to use existing analyses
+* How to write your own analyses
+* Parallel analysis in an HPC setting
+* Advanced functionality such as universe construction and transformations
+
+The workshop is aimed at users in academia and industry who are already familiar with Python and molecular dynamics. 
+As many people may have different levels of experience with MDAnalysis, we will have independent sessions ranging from introductory to advanced use.
```

Comment:
Rewrote @lilyminium 's original sentence to avoid "separate sessions" because we do not have individual per-session registration, only full workshop.

Reply:
I'm not sure what you are trying to say here -- if people sign up for all the sessions at once, isn't this functionally a crash course aimed at bringing introductory users up to advanced?